integration/storagetest/maptests.go

```diff
@@ -129,3 +135,53 @@ func (*mapTests) TestMapLayout(ctx context.Context, t *testing.T, s storage.MapS
 		t.Errorf("Layout(): %v", err)
 	}
 }
+
+func (*mapTests) TestGetTile(ctx context.Context, t *testing.T, ms storage.MapStorage, as storage.AdminStorage) {
```

Comment:
This tests a lot more than the name implies. TestSetGetTile or TestTileRoundTrip may be closer to the truth?

Reply:
yep